launcher/Application.cpp

```diff
@@ -696,6 +696,9 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
         // Minecraft launch method
         m_settings->registerSetting("MCLaunchMethod", "LauncherPart");
 
+        // Minecraft offline player name
+        m_settings->registerSetting("LastOfflinePlayerName", "");
```

Comment:
do we need this? We have separate offline accounts option, so it seems redundant. Maybe Launch Offline should be removed entirely
For now it's fine though

Reply:
Yeah for now it's fine, but for some users it's better to just click "play offline" and having an offline session